src/Mapper/ProgrammesDbToDomain/AtozTitleMapper.php

```diff
@@ -38,6 +38,6 @@ private function getCoreEntityModel(array $dbAtozTitle, string $key = 'coreEntit
             throw new DataNotFetchedException('All AtozTitles must be joined to a CoreEntity');
         }
 
-        return $this->mapperFactory->getProgrammeMapper()->getDomainModel($dbAtozTitle[$key]);
+        return $this->mapperFactory->getCoreEntityMapper()->getDomainModelForProgramme($dbAtozTitle[$key]);
```

Comment:
AtoZTitle can contain Programmes or Groups. Now that we've got the common mapper we can now reflect that correctly.